### PR TITLE
Emergency button

### PR DESCRIPTION
**Pull request by wrunion**
Added large, responsive emergency resources button. Linked to the emergency resources page via React Router's <Link> tag. 

This addresses #19

### CODE REVIEW

File: client/src/components/Home/Home.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
 import IconSelector from './IconSelector';
 import SearchBar from './SearchBar';
 import './Home.css';
+import { Link } from 'react-router-dom';
+import BigRedButton from './../../images/bigRedButton.png';
```
Comment (timhitchins) [inline client/src/components/Home/Home.js]: I see what you mean about the margins.  Can we get rid of the other pngs that are not being used?

File: client/src/components/Home/Home.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import PropTypes from 'prop-types';
...
 
 /*---------------------------------------------------------*/
```
Comment (timhitchins) [inline client/src/components/Home/Home.js:9]: I added these I think but we could probably just get rid of them.

File: client/src/components/Home/Home.js [hunk 3]
```diff
@@ -16,7 +18,6 @@ const BasicInstructions = props => {
 };
 
 /*---------------------------------------------------------*/
```
Comment (timhitchins) [inline client/src/components/Home/Home.js:20]: THis one too.

File: client/src/components/Home/Home.js (hunk 1 above)
Comment (wrunion) [reply client/src/components/Home/Home.js]: yep yep

File: client/src/components/Home/Home.js (hunk 2 above)
Comment (wrunion) [reply client/src/components/Home/Home.js:9]: lines 6 and 7? won't the code break?

Comment (wrunion) [reply client/src/components/Home/Home.js:9]: will do

File: client/src/components/Home/Home.js (hunk 3 above)
Comment (wrunion) [reply client/src/components/Home/Home.js:20]: got it

File: config/nodeKeys.js [hunk 4]
```diff
@@ -1,6 +1,6 @@
 //these ids are public facing
 module.exports = {
```
Comment (timhitchins) [inline config/nodeKeys.js:2]: This is the last one that will need to change before merging.  I just need to hear back from sophie.  But as a placeholder, this file will look like this: 
```
if (process.env.NODE_ENV === "production") {
  module.exports = {
    NODE_PACKAGE_ID: "e9c55b2c-4019-463e-8efa-622f23221402",
    NODE_LISTING_RESOURCE: "9be4623a-3c01-4b2d-9c7b-567a41abbc1c",
    NODE_PHONE_RESOURCE: "2f66ad5d-5066-49d8-846a-6751cfd23863",
  };
} else {
  module.exports = {
    NODE_PACKAGE_ID: "592c18db-efa6-44c6-8477-4ffa4103ba94",
    NODE_LISTING_RESOURCE: "61cee891-7d0f-4ebe-b8ea-c0c8d6cb27e7",
    NODE_PHONE_RESOURCE: "4407461b-e99d-4d8e-8a44-18483aa8d13c",
  };
}
```

**Review by timhitchins — APPROVED** [review]
@wrunion thanks for doing this.!  Looks great.  I am going to approve to keep from blocking you.  The only other change that will need to happen before a merge will be to update the `nodeKeys.js` config.  I left an example for it in the comments.

**Comment by wrunion** [comment]
Alright I changed the node keys! Should be ready to go!

**Pull request merged**